azure-samples/src/main/java/com/microsoft/azure/management/network/samples/ManageVpnGatewayPoint2SiteConnection.java

```diff
@@ -0,0 +1,149 @@
+import com.microsoft.azure.management.resources.fluentcore.arm.Region;
+import com.microsoft.azure.management.resources.fluentcore.utils.SdkContext;
+import com.microsoft.azure.management.samples.Utils;
+import com.microsoft.rest.LogLevel;
+
+import java.io.File;
+
+/**
+ * Azure Network sample for managing virtual network gateway.
+ *  - Create a virtual network with subnets
+ *  - Create virtual network gateway
+ *  - Update virtual network gateway with Point-to-Site connection configuration
+ *  - Generate and download VPN client configuration package. Now it can be used to create VPN connection to Azure.
+ *  - Revoke a client certificate
+ *
+ *  Please note: in order to run this sample, you need to have:
+ *   - pre-generated root certificate and public key exported to $CERT_PATH file
+ *      For more details please see https://docs.microsoft.com/en-us/azure/vpn-gateway/vpn-gateway-certificates-point-to-site for PowerShell instructions
+ *      and https://docs.microsoft.com/en-us/azure/vpn-gateway/vpn-gateway-certificates-point-to-site-makecert for Makecert instructions.
+ *   - client certificate generated for this root certificate installed on your machine.
+ *      Please see: https://docs.microsoft.com/en-us/azure/vpn-gateway/point-to-site-how-to-vpn-client-install-azure-cert
+ *   - thumbprint for client certificate saved to $CLIENT_CERT_THUMBPRINT
+ */
+
+public final class ManageVpnGatewayPoint2SiteConnection {
+
+    /**
+     * Main function which runs the actual sample.
+     * @param azure instance of the azure client
+     * @return true if sample runs successfully
+     */
+    public static boolean runSample(Azure azure) {
+        final String certPath = System.getenv("CERT_PATH");
+        final String clientCertThumbprint = System.getenv("CLIENT_CERT_THUMBPRINT");
+        final Region region = Region.US_WEST2;
+        final String rgName = SdkContext.randomResourceName("rg", 20);
+        final String vnetName = SdkContext.randomResourceName("vnet", 20);
+        final String vpnGatewayName = SdkContext.randomResourceName("vngw", 20);
+
+        try {
+            //============================================================
+            // Create virtual network with address spaces 192.168.0.0/16 and 10.254.0.0/16 and 3 subnets
+            System.out.println("Creating virtual network...");
+            Network network = azure.networks().define(vnetName)
+                    .withRegion(region)
+                    .withNewResourceGroup(rgName)
+                    .withAddressSpace("192.168.0.0/16")
+                    .withAddressSpace("10.254.0.0/16")
+                    .withSubnet("GatewaySubnet", "192.168.200.0/24")
+                    .withSubnet("FrontEnd", "192.168.1.0/24")
+                    .withSubnet("BackEnd", "10.254.1.0/24")
+                    .create();
+            System.out.println("Created network");
+            // Print the virtual network
+            Utils.print(network);
+
```

Comment:
For consistency with other samples can we add the comment

//============================================================
// Create virtual network gateway

Reply:
Fixed.